sources/packages/api/src/route-controllers/models/common.dto.ts

```diff
@@ -15,14 +15,30 @@ export class AddressAPIInDTO {
   addressLine2: string;
   @IsNotEmpty()
   city: string;
-  @IsNotEmpty()
+  @IsOptional()
```

Comment:
can we please add provinceState? as its optional?

Reply:
mm, I didn't understand @guru-aot